packages/entities/entities-redis-configurations/src/composables/useRedisConfigurationForm.ts

```diff
@@ -211,11 +211,25 @@ export const useRedisConfigurationForm = (options: Options) => {
       form.readonly = true
       form.errorMessage = ''
 
+      let payloadData = payload.value
```

Comment:
Maybe by marking `payloadData` a `Partial<typeof payload` we can avoid the type casting 👇

Reply:
updated